account_oauth_client.go

```diff
@@ -129,3 +129,14 @@ func (c *Client) DeleteOAuthClient(ctx context.Context, clientID string) error {
 	err := doDELETERequest(ctx, c, e)
 	return err
 }
+
+// ResetOAuthClientSecret resets the OAuth Client secret for a client with a specified id
+func (c *Client) ResetOAuthClientSecret(ctx context.Context, clientID string) (*OAuthClient, error) {
```

Comment:
New function to reset oauth client secret - https://techdocs.akamai.com/linode-api/reference/post-reset-client-secret